store.html

```diff
@@ -0,0 +1,164 @@
+                  <option value="london">London</option>
+                  <option value="birmingham">Birmingham</option>
+                  <option value="manchester">Manchester</option>
+                  <option value="guildford">Guildford</option>
+                </select>
+              </div>
+
+              <div class="postcode-wrapper">
+                <label for="postcode">Postcode*</label>
+                <input
+                  class="name"
+                  type="text"
+                  id="postcode"
+                  name="postcode"
+                  required
+                />
+              </div>
+            </div>
+
+            <fieldset>
+              <legend>Select a colour</legend>
+
+              <input
+                type="radio"
+                id="orange"
+                name="karma-orange"
+                value="Karma orange"
+                checked
+              />
+              <label for="orange">Karma Orange</label>
+
+              <input
+                type="radio"
+                id="grey"
+                name="space-grey"
+                value="Space grey"
+              />
+              <label for="grey">Space Grey</label>
+            </fieldset>
+
+            <div class="box-terms">
+              <input type="checkbox" id="terms" name="terms and conditions" />
+              <div class="tick-wrapper">
+                <svg
+                  class="icon"
+                  width="17"
+                  height="17"
+                  viewBox="0 0 17 17"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M15.3624 0.500781L15.3622 0.50079C15.018 0.515038 14.7397 0.712028 14.5632 0.962878L5.80398 13.1336L2.44018 8.45967C2.3505 8.33105 2.23519 8.21634 2.09472 8.13198C1.95137 8.04589 1.78691 7.99482 1.61264 7.99235C1.43827 7.98989 1.27215 8.03635 1.12618 8.11916C0.98155 8.2012 0.862485 8.31531 0.769755 8.44416C0.67702 8.57301 0.609113 8.71844 0.564752 8.86942C0.520316 9.02065 0.498888 9.17939 0.500044 9.33735C0.501201 9.49529 0.52495 9.6537 0.571762 9.80419C0.617958 9.9527 0.687269 10.0955 0.780954 10.2216L4.97435 16.0482L4.97441 16.0483C5.15845 16.3039 5.44947 16.5 5.80398 16.5C6.15848 16.5 6.4495 16.3039 6.63355 16.0483L6.63361 16.0482L16.2225 2.72471C16.3634 2.53308 16.4489 2.30339 16.4829 2.07016C16.5171 1.83492 16.5001 1.58974 16.4309 1.36263C16.3621 1.13681 16.239 0.920931 16.057 0.758695C15.8718 0.593603 15.6297 0.48983 15.3624 0.500781Z"
+                    fill="#DE6E49"
+                    stroke="#DE6E49"
+                  />
+                </svg>
+              </div>
+              <label for="terms"
+                >By placing your order you agree to Karma's<a
+                  href="termsconditions"
+                  >Terms and Conditions</a
+                >*</label
+              >
+            </div>
+
+            <button>Place my order</button>
```

Comment:
Interesting to know:

When inside a `<form>` :
`<input type="submit">Place my order</input>`
Will create a button thats function is to submit the form

And also when inside a `<form>`
a single `<button>` with no defined type will default to "submit"

Whereas in this case:
```
<button type="submit">Submit</button>
<button type="reset">Reset</button>
```
If you have multiple buttons, it is important to explicitly name their type.

So yours would be:
`<button type="submit">Place my order</button>`

I know this does not apply in this case (because there is only a single button) but it is good to know for later on.